bindings/rust/standard/integration/src/features/mod.rs

```diff
@@ -2,4 +2,4 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #[cfg(feature = "pq")]
-mod mldsa;
+mod pq;
```

Comment:
This is probably fine for a temporary hack, but it seems sad that we have an entire "features" folder but we're going to shove all the features into one module. If we had a third feature that wasn't pq, what would we rename this module to? "misc"? The more correct way to handle this would probably be to move any code you need to reuse out of mldsa so you can have a separate mlkem file specifically for mlkem tests.